### DISCUSSION TITLE
urgencia

### DISCUSSION THREAD

**Discussion by zimba64** [BIPES/BIPES#221] [Q&A] [answered]
hola eu sou o zimba64
eu trabalho com eletronica a mais de 40 anos .
sempre me interecei por programacao des da epoca que concertava apleII Na santa ifigenia  .
mas sou dislexico e pontos e virgula me impedem de programar . 
mas eu vi esta plataforma BIPES e me impolguei  
eu consegui faser o programa pisca   
mas nao concigo jogalo para funcionar independente desligado do computador  
alguem poderia me ajudar  .

**Comment by joaodrj** [comment, marked as answer]
Para que los programas se ejecuten en la plataforma bipes, debe guardar el archivo con main.py en el administrador de archivos en la pestaña Archivos, cambiar el nombre code.bipes.py a "main.py y hacer clic en Guardar una copia y se ejecutará. trabajar normalmente fuera de la plataforma.
